cparser/processor.js

```diff
@@ -53,6 +61,10 @@ module.exports = that = {
 
 			var includeStr = '#include "application.h"';
 			var appDotHInclude = fileBuffer.indexOf(includeStr);
+			if (appDotHInclude === -1) {
```

Comment:
I can switch this to look for application.h, Particle.h and Arduino.h. Is this what you are saying?